README.md

```diff
@@ -0,0 +1,6 @@
+Hello, World
```

Comment:
Was this done as a git rename? I believe if you had moved it with `git mv README README.md`, it would show a more useful diff. Can you see if you can make it a rename?

Reply:
Umm nope